content/get-started/get-started.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Getting Started
+seo-title: Getting Started
+---
+
+Iron Fish is cryptocurrency that allows for safe, secure, and private transactions.
+
+This guide is intended for crypto novices to understand what Iron Fish is and how they can get started using it. If you want to skip directly to installing and running Iron Fish, you can [install a node](/use/get-started/installation) now.
+
+The currency symbol for Iron Fish is $IRON. This is compared to United States Dollar (USD), Chinese Yuan (CNY), or Russian Ruble (RUB).
+
+### Why use $IRON?
+
+Currencies like $IRON promote decentralization by creating a global ledger that no single person can modify. When you run Iron Fish, you help keep this ledger secure by checking that transactions are following the balances in the ledger. By running Iron Fish, you help keep your and your communities coins secure.
+
+The advantage of $IRON is that it's fundamentally private. Other people cannot see the amount sent, or the recipient without your permission. That's thanks to technology called Zero Knowledge Proofs. This is different from Bit Coin which is fundamentally public.
+
+### How can I use $IRON?
+
+There are three way Fish network. Choosing which one is the right solution for you depends on your goals and technical skill.
+
+#### Wallets
+
+Wallets are a simple easy to use way to send and recieve $IRON.
+
+They allow you to send and recieve coins. When you use a wallet, other Iron Fish users will validate your transactions before they are added to the ledger.
+
+[Download a supported wallet.](use/ecosystem?category=wallets)
+
+#### Mining Blocks
+
+Mining means taking transactions, and adding them to blocks. When mining is done by many different people all over the world, it's one of the main ways Iron Fish remains decentralized and secure. Miners can receive receive $IRON as a reward.
```

Comment:
More opinionated suggestion: 

```suggestion
Mining plays a pivotal role because it ensures Iron Fish's decentralization and security. When people from various locations engage in mining, it prevents any single entity from gaining too much control and safeguards the system against potential attacks or manipulation. In essence, mining helps distribute power and maintain the integrity of Iron Fish.
```

Reply:
Words like pivotol are not normal english words. They are technical words that are not in peoples vernacular.

Reply:
I think reading this, it just sounds too technical too me and written by an engineer. Can someone with english as a second language that's starting out really understand this? Can non technical people understand this?

Reply:
Got it